docs/learn/avalanche/avalanche-consensus.md

```diff
@@ -2,44 +2,77 @@
+that ensures that everyone sees the same information and that the information is accurate and trustworthy.
 
-Consensus is the task of getting a group of nodes to come to an agreement on a decision.
-Nodes can reach a consensus by following a set of steps called a consensus protocol. Avalanche
-is a new consensus protocol that is scalable, robust, and decentralized. It has low latency and high
-throughput. It is energy efficient and does not require special computer hardware. It performs well
-in adversarial conditions and is resilient to "51% attacks." This document explains the Avalanche
-consensus protocol. The white-paper is [here](https://www.avalabs.org/whitepapers).
+**Avalanche Consensus** is a new consensus protocol that is scalable, robust, and decentralized. It
+combines features of both classical and Nakamoto consensus mechanisms to achieve high throughput,
+fast finality, and energy efficiency. For the whitepaper, see
+[here](https://www.avalabs.org/whitepapers). 
+
+Key Features Include: 
+
+- Speed: Avalanche consensus provides sub-second, immutable finality, ensuring that transactions are
+quickly confirmed and irreversible.
+- Scalability: Capable of high throughput with a latency of less than half a second.
+- Energy Efficiency: Unlike other popular consensus protocols, participation in Avalanche consensus
+is not computationally intensive nor expensive. 
+- Adaptive Security: Because the security level of the network adjusts depending on the number of Byzantine
+nodes (bad actors), the network is more resilient to attacks. 
 
 ![Consensus Comparison](/img/Consensus-protocol-comparison.png)
 
-Protocols in the Avalanche family operate through repeated sub-sampled voting. When a
+## Conceptual Overview
+
+Consensus protocols in the Avalanche family operate through repeated sub-sampled voting. When a
 [validator](http://support.avalabs.org/en/articles/4064704-what-is-a-blockchain-validator) is
 determining whether a
 [transaction](http://support.avalabs.org/en/articles/4587384-what-is-a-transaction) should be
-accepted, it asks a small, random subset of validators on their preferences. Each queried validator
-replies with the transaction that it prefers, or thinks should be accepted. The transaction that is
-received from the reply might be the same transaction indicating agreement, or it might be another
-conflicting transaction.
+accepted, it asks a small, random subset of validators for their preference. Each queried validator
+replies with the transaction that it prefers, or thinks should be accepted.
+
+If a majority of the validators sampled reply with the same preferred
+transaction, this becomes the preferred choice of the group sampled, including the validator that asked.
+In the future, this validator and all those who have participated in at least one round of sampling
+will reply with the transaction preferred by the majority.
```

Comment:
This is not quite accurate. Does this need to be changed in this PR? The previous version was not quite correct either.

Votes are performed through vertices/blocks as opposed to transactions.

Additionally, validators maintain a confidence counter and do not immediately change their preference after a single poll in favor of a different container (transaction in the current text).